raspi/soundserver/soundserver.py

```diff
@@ -153,6 +157,58 @@ def restore_hardvolume_route():
     vlcplayer.restore_hardvolume()
     return do_return('Ok', 200)
 
+@app.route('/reset_smart_speaker/<type>', methods=['POST','PUT'])
```

Comment:
Hmmm, I am not really happy to have reset functionality in the **sound server** ...

Furthermore, I would prefer if this functionality is exposed only when authenticated, but not by default.

Reply:
@norbusan thank you for reviewing. 
As discussed in the last meeting, @mariobehling asked to integrate the susi.ai WebClient in the smart speaker and add this page to the devices section, which will then be used to control the smart speaker.

So I suggest that all things which are concerned about controlling the device can be added in this flask server and it should be renamed to **control server**. I agree with not having things like reset options exposed. I can work on having some options available only when a password is set after this PR is merged.

Reply:
Ok, fine with me. How do we proceed? Should I merge this, then rename and adjust systemd etc? Or should the renaming be done first and then we work on this?

Reply:
Yes, let us merge it now and we'll rename it afterwards.
I can take up the task of renaming and adding changes to the server so that reset features are hidden until a password is set. I'll open an issue for these things as well after the merge.

Reply:
@aker99 please comment out the reset endpoint for now.